docroot/modules/custom/uiowa_core/uiowa_core.libraries.yml

```diff
@@ -2,6 +2,21 @@ gtag:
   version: 1.x
   js:
     js/tag-manager.js: {}
+graph:
+  version: 1.x
+  js:
+    js/csv_graph.js: {}
+  dependencies:
+    - core/jquery
+    - core/jquery.once
+    - core/drupalSettings
```

Comment:
I'd be curious if adding a dependency here on `uiowa_core/chartjs` solved the loading issue.

Reply:
like, depending on another library in the libraries file?

Reply:
testing now

Reply:
Yea IDK if you can depend on a library that you define in your own library file. But I assume Drupal would handle loading these accordingly if so.

Reply:
it did not